geopyspark/geotrellis/constants.py

```diff
@@ -93,237 +44,200 @@
-INT32RAW = "int32raw"
-
-"""Representes Float Cells."""
-FLOAT32RAW = "float32raw"
-
-"""Representes Double Cells."""
-FLOAT64RAW = "float64raw"
-
-"""Representes Bit Cells."""
-BOOL = "bool"
-
-"""Representes Byte Cells with constant NoData values."""
-INT8 = "int8"
-
-"""Representes UByte Cells with constant NoData values."""
-UINT8 = "uint8"
-
-"""Representes Short Cells with constant NoData values."""
-INT16 = "int16"
-
-"""Representes UShort Cells with constant NoData values."""
-UINT16 = "uint16"
-
-"""Representes Int Cells with constant NoData values."""
-INT32 = "int32"
-
-"""Representes Float Cells with constant NoData values."""
-FLOAT32 = "float32"
-
-"""Representes Double Cells with constant NoData values."""
-FLOAT64 = "float64"
-
-CELL_TYPES = [
-    BOOLRAW,
-    INT8RAW,
-    UINT8RAW,
-    INT16RAW,
-    UINT16RAW,
-    INT32RAW,
-    FLOAT32RAW,
-    FLOAT64RAW,
-    BOOL,
-    INT8,
-    UINT8,
-    INT16,
-    UINT16,
-    INT32,
-    FLOAT32,
-    FLOAT64
-]
-
-
-"""A ColorRamp."""
-HOT = "hot"
-
-"""A ColorRamp."""
-COOLWARM = "coolwarm"
-
-"""A ColorRamp."""
-MAGMA = "magma"
-
-"""A ColorRamp."""
-INFERNO = "inferno"
-
-"""A ColorRamp."""
-PLASMA = "plasma"
-
-"""A ColorRamp."""
-VIRIDIS = "viridis"
-
-"""A ColorRamp."""
-BLUE_TO_ORANGE = "BlueToOrange"
-
-"""A ColorRamp."""
-LIGHT_YELLOW_TO_ORANGE = "LightYellowToOrange"
-
-"""A ColorRamp."""
-BLUE_TO_RED = "BlueToRed"
-
-"""A ColorRamp."""
-GREEN_TO_RED_ORANGE = "GreenToRedOrange"
-
-"""A ColorRamp."""
-LIGHT_TO_DARK_SUNSET = "LightToDarkSunset"
-
-"""A ColorRamp."""
-LIGHT_TO_DARK_GREEN = "LightToDarkGreen"
-
-"""A ColorRamp."""
-HEATMAP_YELLOW_TO_RED = "HeatmapYellowToRed"
-
-"""A ColorRamp."""
-HEATMAP_BLUE_TO_YELLOW_TO_RED_SPECTRUM = "HeatmapBlueToYellowToRedSpectrum"
-
-"""A ColorRamp."""
-HEATMAP_DARK_RED_TO_YELLOW_WHITE = "HeatmapDarkRedToYellowWhite"
-
-"""A ColorRamp."""
-HEATMAP_LIGHT_PURPLE_TO_DARK_PURPLE_TO_WHITE = "HeatmapLightPurpleToDarkPurpleToWhite"
 
-"""A ColorRamp."""
-CLASSIFICATION_BOLD_LAND_USE = "ClassificationBoldLandUse"
-
-"""A ColorRamp."""
-CLASSIFICATION_MUTED_TERRAIN = "ClassificationMutedTerrain"
-
-COLOR_RAMPS = [
-    HOT,
-    COOLWARM,
-    MAGMA,
-    INFERNO,
-    PLASMA,
-    VIRIDIS,
-    BLUE_TO_ORANGE,
-    LIGHT_YELLOW_TO_ORANGE,
-    BLUE_TO_RED,
-    GREEN_TO_RED_ORANGE,
-    LIGHT_TO_DARK_SUNSET,
-    LIGHT_TO_DARK_GREEN,
-    HEATMAP_YELLOW_TO_RED,
-    HEATMAP_BLUE_TO_YELLOW_TO_RED_SPECTRUM,
-    HEATMAP_DARK_RED_TO_YELLOW_WHITE,
-    HEATMAP_LIGHT_PURPLE_TO_DARK_PURPLE_TO_WHITE,
-    CLASSIFICATION_BOLD_LAND_USE,
-    CLASSIFICATION_MUTED_TERRAIN
-]
+class ResampleMethod(Enum):
+    """Resampling Methods."""
+
+    NearestNeighbor = 'NearestNeighbor'
+    Bilinear = 'Bilinear'
+    CubicConvolution = 'CubicConvolution'
+    CubicSpline = 'CubicSpline'
+    Lanczos = 'Lanczos'
+    Average = 'Average'
+    Mode = 'Mode'
+    Median = 'Median'
+    Max = 'Max'
+    Min = 'Min'
+
+    RESAMPLE_METHODS = [
```

Comment:
As mentioned above, we'd like to keep the option of letting the user input a string instead of constant if they so desire. That's the main reason for keeping those lists around, as `"NearestNeighbor" in ResampleMethod` will always return `False`.